README.md

```diff
@@ -62,7 +62,9 @@ Please refer to [tfvars collection](/tfvars_collection) for more examples, i.e.
   "tsb_org": "tetrate",
   "aws_k8s_regions": [],
   "azure_k8s_regions": ["eastus"],
-  "gcp_k8s_regions": ["us-west1", "us-east1"]
+  "gcp_k8s_regions": ["us-west1", "us-east1"],
+  "tetrate_owner": "username",
+  "tetrate_team": "function:team"
```

Comment:
added to prereqs under README.md

Reply:
[nit] Since it is important that we enforce the right values here, would it be better to add a value like:

```
"tetrate_owner": "Change me! (URL)",
```

To avoid an easy copy/paste mistake? In AWS policies can't validate the _value_ of the tags, so we should try to minimize the chances invalid values are set just because of copy/paste.